Server/src/services/validationSchema.ts

```diff
@@ -0,0 +1,113 @@
+//Running and installing [npm i jsonschema] is required for this page to work
+//More details and examples about json schema validator can be found at:
+// https://github.com/tdegrunt/jsonschema#readme
+
```

Comment:
Is this duplicated code?

Reply:
No, this is the instructions to make sure the json schema validator is installed before you run the file, "npm i jsonschema" is required to run it correctly